components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -872,6 +872,7 @@ void RewardsServiceImpl::OnReconcileComplete(ledger::Result result,
       return;
 
     FetchWalletProperties();
+    MaybeShowNotificationTipsPaid();
```

Comment:
we need to check for `ledger::REWARDS_CATEGORY` if it's recurring tip and only then call this function

Reply:
done